privacy-protections/click-to-load/main.js

```diff
@@ -64,6 +65,9 @@ function facebookObserver (list, observer) {
     }
 
     updateResults();
+    if (loadsUpdated) {
```

Comment:
Just an idea, feel free to ignore. But we could put this check inside of `updateLoads`. That way, we could display "DETECTED" right away when a request was detected (even before they all finish), but still wait until the end to display "NONE".

Reply:
Tweaked